lib/templates_helpers/at_reCaptcha.js

```diff
@@ -0,0 +1,19 @@
+AT.prototype.atReCaptchaRendered = function() {
+    $.getScript('https://www.google.com/recaptcha/api.js');
+};
+
+AT.prototype.atReCaptchaHelpers = {
+    key: function() {
+        if (Meteor.settings.public.reCaptcha && Meteor.settings.public.reCaptcha.siteKey)
+            return Meteor.settings.public.reCaptcha.siteKey;
+        return AccountsTemplates.options.reCaptcha.siteKey;
+    },
```

Comment:
This one is better than the other, but still nothing guarantees that the value is present...